pkg/storage/accountsDB/accountDB.go

```diff
@@ -0,0 +1,99 @@
+package accountsDB
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+	"sync"
+)
+
+type AccountStorage struct {
+	MapAccount map[uuid.UUID]model.Account
+	mu         sync.Mutex
+}
+
+func NewAccountStorage() *AccountStorage {
+	as := AccountStorage{}
+	as.MapAccount = make(map[uuid.UUID]model.Account)
+	return &as
+}
+
+func (as *AccountStorage) GetAccount(_ context.Context, id uuid.UUID) (model.Account, error) {
+	as.mu.Lock()
+	val, ok := as.MapAccount[id]
+	as.mu.Unlock()
+	if !ok {
+		return model.Account{}, fmt.Errorf("account with id %d not found", id)
```

Comment:
Return error or empty value without error. Other DB can return many type of errors and handle empty data can be difficult.

Reply:
Do you mean to add custom error type?

Reply:
PS. Change all occurances

Reply:
Or custom error type or when function return nil, nil(value, error) - can`t find.